### DISCUSSION TITLE
Move to MAUI Community Toolkit

### DISCUSSION THREAD

**Discussion by maxkoshevoi** [thudugala/Plugin.LocalNotification#194] [Q&A]
Hi, starting from Xamarin 5 there is [XamarinCommunityToolkit](https://github.com/xamarin/XamarinCommunityToolkit) NuGet package that is supported by community and intended to be single stop for all Xamarin needs that isn't included in official NuGets. 

Are there plans to add this package there?

**Comment by thudugala** [comment]
Yes, when time permit

**Comment by LeoJHarris** [comment]
@thudugala can you shed any light on when this might be performed?

**Comment by thudugala** [comment]
@LeoJHarris before getting started, I'm still waiting on version 9 to finalize and also need to write unite test.

**Reply by thudugala** [reply]
@LeoJHarris thank you very much

**Comment by maxkoshevoi** [comment]
XamarinCommunityToolkit won't be accepting any new features starting September. Renamed the discussion to "Move to MAUI Community Toolkit"

**Reply by LeoJHarris** [reply]
@maxkoshevoi Interesting but once it migrates to the MAUI Community Toolkit they will accept PR on new features?

**Reply by maxkoshevoi** [reply]
Yes, but it will be a different NuGet that will target .net6

Here's more info about this whole thing: https://devblogs.microsoft.com/xamarin/the-future-of-xamarin-community-toolkit/

**Reply by LeoJHarris** [reply]
@maxkoshevoi Cool thanks for the heads up on that.
